rest-server/core/src/main/scala/com/actionml/core/spark/SparkContextSupport.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright ActionML, LLC under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * ActionML licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.actionml.core.spark
+
+import cats.data.Validated
+import cats.data.Validated.{Invalid, Valid}
+import com.actionml.core.validate._
+import com.mongodb.spark.MongoSpark
+import com.mongodb.spark.rdd.MongoRDD
```

Comment:
The Spark context creation should not be coupled to Mongo.

Reply:
Mongo and Elasticsearch clients should consume the context and even provide key/value pairs for creating the context but no method of the SparkContextSupport should assume Mongo or Elasticsearch.